container/root/run.d/.gitkeep

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
Just double checking, this file is suppose to be a bash script? I thought `.gitkeep` is just for making sure blank directories get committed?